.github/workflows/build-test-release.yml

```diff
@@ -36,6 +39,7 @@ jobs:
         run: |
           pnpm run lint
           pnpm run check-format
+          pnpm run check-changeset
```

Comment:
nitpick (feel free to not change): In other repos, I've done `pnpm exec changeset status` rather than adding a script to the package.json, since it's unlikely that any code except CI would need to run this.

Reply:
- Added @changesets/changelog-github with the corresponding configuration
- I created the script in package.json for symmetry with the other check command, but I am happy to change it according to your suggestion (done)